src/agent/Makefile

```diff
@@ -27,6 +27,15 @@ COMMIT_MSG = $(if $(COMMIT),$(COMMIT),unknown)
 # Exported to allow cargo to see it
 export VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+##VAR SECCOMP=yes|no define if agent enables seccomp feature
+SECCOMP := yes
+
+# Enable seccomp feature of rust build
+SECCOMP_FEATURE :=
+ifeq ($(SECCOMP),yes)
+    override SECCOMP_FEATURE := --features seccomp
```

Comment:
Might be worth making this `EXTRA_FEATURES` to be more future-proof.

Reply:
I changed `SECCOMP_FEATURE` to `EXTRA_RUSTFEATURES`.